be/src/io/cache/block_file_cache.cpp

```diff
@@ -1579,6 +1700,7 @@ std::string BlockFileCache::reset_capacity(size_t new_capacity) {
                 ss << " ttl_queue released " << queue_released;
             }
             _disk_resource_limit_mode = true;
+            _disk_limit_mode_metrics->set_value(1);
```

Comment:
consider setting it to percentage of utilization of the disk if it enterred this mode.
`_disk_limit_mode_metrics` -> `_disk_size_limit_`
and we can also monitor the size utilization and the inode untilization respectively.